examples/source/source.ts

```diff
@@ -30,13 +30,13 @@ class Sourcer implements source.Sourcer {
     async ack(offsets: source.Offset[]): Promise<void> {
         console.log('ack')
         for (const offset of offsets)
-            console.log("Offset: " + offset.offset.toString() + " Partition ID: " + offset.partitionId)
```

Comment:
nit: Use the backtick string interpolation syntax for all `console.log` statements.